rush-plugins/rush-phased-install-plugin/src/operationRunners/AuthenticateOperationRunner.ts

```diff
@@ -86,7 +87,16 @@ export class AuthenticateOperationRunner implements IOperationRunner {
     };
 
     // eslint-disable-next-line require-atomic-updates
-    tarball.storageUrl = await this._fetchInternal(options, 3);
+    // tarball.storageUrl = await this._fetchInternal(options, 3);
+
+    // to support registry that no need auth
+    const registryUrlWithoutAuth: string = 'https://registry.npmjs.org/';
+
+    const tarballUrl: string = getNpmTarballUrl(packageName, version, {
```

Comment:
Does the tarball download directly from the registry domain? If that's the case ideally you strip out the `_phase:authenticate` altogether and jump straight to `_phase:download`

Reply:
Yeah, seems for public registry domain no need any auth or redirects. So `_phase:authenticate`  only for auth and redirects, I believe we can remove this phase